### PR TITLE
Server Docker Container Build

### PR DESCRIPTION
**Pull request by nickdelgrosso**
Hi,  

This pull request has a working dockerfile for the server component, building on the work done in the coding club session on Friday.  It also includes a docker compose file with the `docker run` settings included, so getting it built and running with ports mapped is just

```
cd src/server
docker compose up
```

I hope this is helpful.

Best wishes,

Nick

### CODE REVIEW

File: src/server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN apt-get update 
+RUN apt-get -y install build-essential libaec-dev libblosc-dev
+
+#RUN conda init bash
+# Make RUN commands use the new environment:
+RUN conda create -n dcp-env python=3.9
+RUN echo "conda activate dcp-env" >> ~/.bashrc
+SHELL ["/bin/bash", "--login", "-c"]
+
```
Comment (hpelin) [inline src/server/Dockerfile:13]: added installation from conda because of missing image codex dependencies

File: src/server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM continuumio/miniconda3
+
+COPY . .
```
Comment (hpelin) [inline src/server/Dockerfile:3]: separate copying the requirements.txt and other files

```suggestion
COPY ./requirements.txt .
```

**Review by hpelin — COMMENTED** [review]
Looks good, added minor changes, it is working.

**Comment by christinab12** [comment]
Nice thanks everyone :)

**Pull request merged**